lib/internal/mime.js

```diff
@@ -122,13 +122,29 @@ const encode = (value) => {
 };
 
 class MIMEParams {
-  #data = new SafeMap();
+  #data = null;
+  #processed = false;
+  #string = null;
+
+  constructor() {
+    this.#data = new SafeMap();
+    this.#processed = true;
```

Comment:
Can you add comments on why you are using a constructor here? It would seem to me that we would be better off without it.